SEP-0002-revised.md

```diff
@@ -0,0 +1,245 @@
+
+* **Membership Limit**: The board shall consist of at least 5 members and no more than 10 members.
+Within these limits, the board may increase or decrease the number of people serving on the board, including for the purpose of staggering the terms of members.
+
+* **Term Length**: All members shall be elected to serve a maximum of a two-year term, however the term may be extended until a successor has been elected. 
+A member may be re-elected at the end of their term.
+
+* **Staggering**: Members terms shall be staggered so that approximately half the number of members will end their terms in any given year.
+The term of office shall be considered to begin January 1 and end December 31 of the second year in office, unless the term is extended until such time as a successor has been elected.
+
+* **Vacancies**: The board may fill vacancies or may appoint new members to fill a previously unfilled board position, subject to the maximum number of members.
+The board shall fill vacancies as soon as possible.
+The board should work to ensure that the composition of the board represents a diverse cross-section of the solar physics community.
+This diversity should consider things such as research topic, career stage, as well as all other forms of diversity such as gender.
+
+* **Removal**: A board member may be removed by a two-thirds majority vote of the full board, if (a) the member is absent and unexcused from 50% or more meetings of the board in a twelve month period.
+The board chair is empowered to excuse members from attendance for a reason deemed adequate by the board chair.
+The chair shall not have the power to excuse him/herself from the board meeting attendance and in that case, the board vice chair shall excuse the chair.
+Or (b) for cause or no cause, if before any meeting of the board at which a vote on removal will be made the director in question is given electronic or written notification of the board’s intention to discuss her/his case and is given the opportunity to be heard at a meeting of the board.
+The board can remove the executive director by a two-thirds majority vote of the full board for cause or no cause.
+
+* **Meetings**:  The board shall have at least a meeting every quarter at times and places fixed by the board.
+Board meetings shall provide at least one week (7) days notice.
+Notice of meetings shall specify the place, day, and hour of meeting.
+The purpose of the meeting need not be specified.
+
+* **Board officers**: The officers of the board shall be a board chair, vice-chair, and secretary, all of whom shall be chosen by, and serve at the pleasure of, the board.
+Officers shall serve one year terms and can serve consecutive terms.
```

Comment:
Is this a one year term on the board, or a one year term as an officer (within a two year term on the board)?

Reply:
It's in the officers role, I have made this explicit.